crossbuilder

```diff
@@ -778,11 +779,18 @@ if [ -z "$COMMAND" ] ; then
     detect_package
     check_changelog_exists
     variables
-    echo "${POSITIVE_COLOR}Building $PACKAGE for $TARGET_ARCH and deploying to device.${NC}"
+    if [ "$DEPLOY" ] ; then
```

Comment:
remove the space before the `;`

Reply:
I just wanted to keep it consistent with the other if's, see above.

Reply:
Ah! Then ignore this :-)